web/src/views/NewIdea.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="container-app  container-extra-padding">
+    <div class="form-section">
+      <label class="form-section__label text--secondary" for="creator">Your name</label>
+      <input
+        class="form-section__input"
+        v-model="creator"
+        type="text"
+        id="creator">
+    </div>
+    <div class="form-section">
```

Comment:
A little spacing (0.4rem) between the label and the input would look nice?

Reply:
The labels already have a bottom margin of 0.4rem because of the class "form-section__label". Should that margin be increased?

Reply:
I see. 0.8rem then. If still too tight (I doubt it'd be), try 1.6rem.

Reply:
0.8rem looks good to me. I have pushed it